internal/controllers/orano2ims_controller.go

```diff
@@ -224,6 +222,108 @@ func (r *ORANO2IMSReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 	return
 }
 
+func (r *ORANO2IMSReconciler) createClientServiceAccount(ctx context.Context,
+	orano2ims *oranv1alpha1.ORANO2IMS) error {
+	_, err := r.createServiceAccount(ctx, orano2ims, utils.ORANO2IMSClientSAName)
+	return err
+}
+
+func (r *ORANO2IMSReconciler) createDeploymentManagerServiceAccount(ctx context.Context,
+	orano2ims *oranv1alpha1.ORANO2IMS) error {
+	// Create the service account:
+	sa, err := r.createServiceAccount(ctx, orano2ims, utils.ORANO2IMSDeploymentManagerServerName)
+	if err != nil {
+		return err
+	}
+
+	// The role and binding are cluster level objects, so to make them unique we use the
+	// namespace as a prefix:
+	roleName := fmt.Sprintf("%s-%s", orano2ims.Namespace, utils.ORANO2IMSAlarmSubscriptionServerName)
+	bindingName := fmt.Sprintf("%s-%s", orano2ims.Namespace, utils.ORANO2IMSAlarmSubscriptionServerName)
+
+	// Grant the service account permissions to read the objects it needs, for example managed
```

Comment:
We could consider moving the creation of the ClusterRole & ClusterRoleBinding to their own functions, like for the other resources create by the operator.

Reply:
Yes, makes sense, will do it.